resources/user-guides/restore_from_keys.md

```diff
@@ -0,0 +1,26 @@
+---
+layout: static_page
+title: "Restoring a wallet from private keys"
+---
+
+
+
+Restoring a wallet from private keys (via the command line in this instance) is pretty simple. If you have the necessary information, with this guide you can completely restore your wallet. Note, you do NOT have to have your password to restore from keys.
```

Comment:
Thanks for the suggestions, all implemented except for this

> Capitalize

Before that is a comma though, not a period...